test/tests/CMakeLists.txt

```diff
@@ -117,8 +117,13 @@ target_link_libraries(test_string ${GTEST_LIBRARIES} test-common dl z)
 add_test(NAME string COMMAND test_string)
 
 # Ride ratings test
-set(RIDE_RATINGS_TEST_SOURCES "${CMAKE_CURRENT_LIST_DIR}/RideRatings.cpp"
-                              "${CMAKE_CURRENT_LIST_DIR}/TestData.cpp")
-add_executable(test_ride_ratings ${RIDE_RATINGS_TEST_SOURCES})
-target_link_libraries(test_ride_ratings ${GTEST_LIBRARIES} libopenrct2 dl z)
-add_test(NAME ride_ratings COMMAND test_ride_ratings)
+if (NOT "z$ENV{CI}" STREQUAL "z")
+    message("Detected CI environment. Disabling ride rating test.")
+else ()
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink "${CMAKE_CURRENT_LIST_DIR}/testdata/parks" "${CMAKE_CURRENT_BINARY_DIR}/parks")
```

Comment:
Yeah, but it still fails because I forgot I run all these tests in docker which has no `CI` env variable set…